README.md

```diff
@@ -342,6 +342,9 @@ as follows:
 }
 ```
 
+If used as a NixOS module, the home-manager library is available under `lib.hm`.
```

Comment:
As far as I know, this is not the case, `lib.hm` is only available within the context of user's home-manager configuration.

Reply:
I think the intention here is to expose it so it can be used outside the context of the user.

Reply:
Yeah, the `lib` available to Home Manager modules includes the `hm` attribute. It would not be available to NixOS modules.

And the `lib.hm` set is available to HM modules regardless of whether it is used in a standalone fashion or through its NixOS module.